addon/components/cell.js

```diff
@@ -108,16 +108,17 @@ export default Component.extend(PropTypeMixin, {
   },
 
   @readOnly
-  @computed('bunsenModel', 'bunsenView.cellDefinitions', 'cellConfig')
+  @computed('bunsenModel', 'bunsenView.cellDefinitions', 'cellConfig', 'value')
```

Comment:
Might be safer to use `propagatedValue` here. Otherwise, `isRequired` would fire when it's descendant doesn't need to be re-rendered causing it's descendants to be re-rendered anyway. It might not even make a difference but one additional cell will be re-rendered.